src/D2L.Bmx/WriteHandler.cs

```diff
@@ -70,6 +70,11 @@ bool useCredentialProcess
 			if( !data.Sections.ContainsSection( sectionName ) ) {
 				data.Sections.AddSection( sectionName );
 			}
+			var defaultCredentialsFile = parser.ReadFile( SharedCredentialsFile.DefaultFilePath );
```

Comment:
We should probably check that a file exists at `SharedCredentialsFile.DefaultFilePath` first.